src/server.js

```diff
@@ -193,27 +214,31 @@ api.post("/getChunkedData", (req, res, next) => {
   // This request owns the directory, so clean it up when the request finishes.
   req.rmChunk = true;
 
-  // We always have an graph file
-  const graphFile = req.body.graphFile;
+  //TODO: loop through and process all tracks
```

Comment:
I guess it is going to pass tests to rely on an exact fixed order of tracks, but I don't think we really want to do it that way.